src/server/server.py

```diff
@@ -4278,6 +4382,18 @@ def set_vrx_node(data):
     else:
         logger.error("Can't set VRx {0} to node {1}: Controller unavailable".format(vrx_id, node))
 
+@catchLogExceptionsWrapper
+def emit_pass_record(node, lap_time_stamp, inc_rsepoch_flag = False):
+    '''Emits 'pass_record' message (will be consumed by slave timers in cluster, etc).'''
```

Comment:
I think it will be good to mention explicitly that it is consumed by UI and LiveTime as well, as it reads that cluster is the predominate use case, and the others might get lost/forgotten in the 'etc'.

Reply:
Unless I'm missing something, I don't think that 'pass_record' emit is consumed by the UI.

Reply:
"pass_record" isn't listened for in any file in /src/server/static or /src/server/templates .